### PR TITLE
[kube-proxy] refactor build and bump deps version 

### PR DESCRIPTION
**Pull request by sprait**
## Description

- Refactor build and update dependencies for fix security vulnerabilities.
- Additional fixes after refactor for node-manager/early-oom and kube-dns/sts-pods-hosts-appender-init-container

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: kube-proxy
type: chore
summary: Refactor build and update dependencies for fix security vulnerabilities.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/static'...

File: modules/036-kube-proxy/images/kube-proxy/werf.inc.yaml [hunk 1]
```diff
@@ -2,9 +2,14 @@
...
-artifact: {{ .ModuleName }}/{{ $.ImageName }}-artifact
-from: {{ $.Images.BASE_ALT_DEV }}
+image: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
+fromImage: common/relocate-artifact
+final: false
 shell:
+  beforeInstall:
+    - apt-get update
+    - apt-get install conntrack-tools -y
+    - rm -rf /var/lib/apt/lists/* /var/cache/apt/* && mkdir -p /var/lib/apt/lists/partial /var/cache/apt/archives/partial
```
Comment (sprait) [inline modules/036-kube-proxy/images/kube-proxy/werf.inc.yaml:12]: need apply after merge refactored relocate-artifact in main
```suggestion
    - apt-get install conntrack-tools -y
```

File: modules/036-kube-proxy/images/kube-proxy/werf.inc.yaml [hunk 2]
```diff
@@ -2,9 +2,12 @@
 {{- $iptables_image_version := $iptables_version | replace "." "-" }}
 {{- $binaries := "/usr/sbin/conntrack /usr/lib64/libnetfilter_conntrack.so*" }}
 ---
-artifact: {{ .ModuleName }}/{{ $.ImageName }}-artifact
-from: {{ $.Images.BASE_ALT_DEV }}
+image: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
+fromImage: common/relocate-artifact
```
[the remark this answers is not in the document]
Comment (RomanenkoDenys) [reply modules/036-kube-proxy/images/kube-proxy/werf.inc.yaml:6]: updated

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'e2e/run/static'...

**Review by name212 — APPROVED** [review]

**Pull request merged**